common/src/main/java/io/pravega/common/lang/RandomStringUtils.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common.lang;
+
+import io.pravega.common.hash.RandomFactory;
+import java.util.Random;
+
+public class RandomStringUtils {
```

Comment:
Please check out PR #2065, which was solving this exact issue, but we closed it because Apache Commons has un-deprecated this class.

Reply:
right, I forgot about that.